### PR TITLE
FIx creationg issue , need a usageType at creation

### PR DESCRIPTION
**Pull request by samcake**

### CODE REVIEW

File: libraries/model/src/model/TextureMap.cpp [hunk 1]
```diff
@@ -128,7 +128,7 @@ gpu::Texture* cacheTexture(const std::string& name, gpu::Texture* srcTexture, bo
                 fclose (file);
 
                 //then create a new texture out of the ktx
-                auto theNewTexure = Texture::unserialize(ktx::KTX::create(storage));
+                auto theNewTexure = Texture::unserialize(gpu::TextureUsageType::RESOURCE, ktx::KTX::create(storage));
```
Comment (jherico) [inline libraries/model/src/model/TextureMap.cpp]: why is this not `srcTexture->getUsageType()`?

**Comment by hifi-gustavo** [comment]
Build console output for commit: 2d7ba45667ff00369cfcb429a88d93e216ace21b available at: https://gist.github.com/48b6c36c30a69d7b6d6cbb312187e16b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9700-2d7ba45.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9700-2d7ba45.dmg

File: libraries/model/src/model/TextureMap.cpp (hunk 1 above)
Comment (samcake) [inline libraries/model/src/model/TextureMap.cpp]: doh!

**Comment by hifi-gustavo** [comment]
Build console output for commit: e6c6cd2825bd476958917d72bfd79f62b801b5e1 available at: https://gist.github.com/48b6c36c30a69d7b6d6cbb312187e16b<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9700-e6c6cd2.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9700-e6c6cd2.dmg

**Pull request merged**